### PR TITLE
Wdfn 425 network monitoring page throws error

### PR DESCRIPTION
**Pull request by abriggs-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

WDFN 425 network monitoring page throws error
-----------
Fix for error when attempting to reach networks/monitoring-locations

Description
-----------
The issue is caused by the code expecting the a JSON file with a values
	"properties": {
             "contactInformation": {
                 "contactName": "Internet of Water Product Owners",
		 "contactEmail": "GS-W-iow_product_owners@usgs.gov"
           }		
	}
But is receiving . . .
	"properties": {
		"contactName": "Internet of Water Product Owners",
		"contactEmail": "GS-W-iow_product_owners@usgs.gov"
	}

The code changes add some extra guardrails to ignore the unexpected JSON

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by jkreft-usgs** [comment]
I'll put in a ticket to change the contact information data structure to match all the others.

File: wdfn-server/waterdata/views.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def networks(network_cd):
     if network_cd:
         collection = network_data
         extent = network_data['extent']['spatial']['bbox'][0]
-        narrative = markdown(network_data['properties']['narrative']) if network_data['properties']['narrative'] else None
+        narrative = markdown(network_data['properties']['narrative']) if network_data['properties'].get('narrative') else None
```
Comment (mbucknell) [inline wdfn-server/waterdata/views.py:218]: You could more tersely write:
```
narrative = markdown(network_data['properties'].get('narrative')
```
See: https://www.w3schools.com/python/ref_dictionary_get.asp

**Review by mbucknell — APPROVED** [review]

File: wdfn-server/waterdata/views.py (hunk 1 above)
Comment (jkreft-usgs) [reply wdfn-server/waterdata/views.py:218]: Does `markdown` barf on `None`?  I assumed that was why it was structured that way.

Comment (abriggs-usgs) [reply wdfn-server/waterdata/views.py:218]: lol. I just asked Mary about that. I think that is the case.

Comment (abriggs-usgs) [reply wdfn-server/waterdata/views.py:218]: Okay, seems like Markdown is the issue, so I guess I'll leave it as is.

**Pull request merged**